app/src/App.js

```diff
@@ -1,53 +1,69 @@
 import React from 'react'
 import { useAragonApi } from '@aragon/api-react'
-import { Main, Button } from '@aragon/ui'
+import {
+  Box,
+  Button,
+  GU,
+  Header,
+  IconMinus,
+  IconPlus,
+  Main,
+  SyncIndicator,
+  Tabs,
+  textStyle,
+} from '@aragon/ui'
 import styled from 'styled-components'
 
 function App() {
-  const { api, appState } = useAragonApi()
+  const { api, appState, path, requestPath } = useAragonApi()
   const { count, isSyncing } = appState
-  console.log(count, isSyncing)
+
+  const pathParts = path.match(/^\/tab\/([0-9]+)/)
+  const pageIndex = pathParts ? parseInt(pathParts[1], 10) - 1 : 0
```

Comment:
```suggestion
  const pageIndex = Array.isArray(pathParts) ? parseInt(pathParts[1], 10) - 1 : 0
```

Reply:
Just to be confirm, is this is for readability? Otherwise `String.prototype.match()` only returns `null` when it doesn’t match.

Reply:
Mostly to have a bit more safety; we immediately assume it's an array afterwards and so might as well make this check here to ensure we don't get a NaN.

Reply:
mmm I’m not sure I follow then :thinking: 

`String.prototype.match()` can only return two things AFAIK: an array, or `null`.

Reply:
My thinking around this is that we could place the type check closer to where it's being used, in case this bit is ever separated or made less obvious that `pathParts` was directly the return of a `match()` call.